frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -98,7 +98,7 @@ export const InsightsPopup: React.FC<PrometheusHealthPopupProps> = ({ responses,
         )}
       </div>
       <div className="co-status-popup__section">
-        {!isWaitingOrDisabled && !isError && (
+        {!isWaitingOrDisabled && !isError && !clusterID && (
```

Comment:
Is the logic backwards here? It looks like you're checking `!clusterID` and then using it in the content.

Reply:
@spadgett, sorry, end of the working day, left it out. Fixed :)